packages/sources/coinmetrics/src/transport/price-ws.ts

```diff
@@ -78,9 +91,19 @@ export const handleAssetMetricsMessage = (
   message: WsAssetMetricsMessage,
 ): ProviderResult<WsTransportTypes>[] | undefined => {
   logger.trace(message, 'Got response from websocket')
+
   if ('error' in message) {
     // Is WsAssetMetricsErrorResponse
     logger.error(message, `Error response from websocket`)
+
+    const regex = /'([^']+)'/g
```

Comment:
What exactly is this regex doing? Is it finding each base in the error message? If so, we should name it 'findBaseCurrenciesRegex' or something like that. Otherwise, this PR looks good 👍

Reply:
It would also be helpful to have a comment with the expected error this Regex parses, for future reference!